src/components/SelectColors/SelectColors.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  /**
+   * CSS class of the selectColors.
+   */
+  className: PropTypes.string,
+
+  /**
+   * Array of values to use in the selectColors element.
+   */
+  options: PropTypes.arrayOf(
+    PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.number,
+      PropTypes.shape({
+        value: PropTypes.oneOfType([
+          PropTypes.string,
+          PropTypes.number,
+          PropTypes.arrayOf(
+            PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+          ),
+        ]).isRequired,
+        label: PropTypes.string.isRequired,
+      }),
+    ]),
+  ),
+
+  /**
+   * Function triggered on select's change event.
+   * @param {Event} event The change event object.
+   */
+  onChange: PropTypes.func.isRequired,
+};
+
+const defaultProps = {
+  options: undefined,
+  className: 'tm-selectcolors',
+};
+
+/**
+ * This component displays a simple selectColors HTML element.
+ */
+const SelectColors = function(props) {
```

Comment:
Can import props directly here, with functional component:
const SelectColors = ({ options, className, onChange }) => {

Reply:
I guess I referred to the older one where React.createElement was used. I updated as jsx in this PR.

Reply:
Yes I commented at the same time you commited the update.

Reply:
But it is still relevant to import directly the function delcaration:
'''
const SelectColors = ({ options, className, onChange }) => {
''''
instead of 
'''
const SelectColors = props => {
  const { options, className, onChange } = props;
'''

Reply:
Done, thanks for the suggestion.